docs/site/angular-auth-oidc-client/docusaurus.config.js

```diff
@@ -81,7 +84,13 @@ module.exports = {
         docs: {
           sidebarPath: require.resolve('./sidebars.js'),
           // Please change this to your repo.
-          editUrl: 'https://github.com/damienbod/angular-auth-oidc-client',
+          editUrl:
+            'https://github.com/damienbod/angular-auth-oidc-client/edit/main/docs/site/angular-auth-oidc-client',
```

Comment:
I noticed the current edit functionally redirects the user to a 404, e.g. https://github.com/damienbod/angular-auth-oidc-client/docs/documentation/public-api.md

Reply:
Nice catch!